tests/browser/cache-first/webpack.config.js

```diff
@@ -3,9 +3,11 @@ const { resolve, join } = path;
 
 const OfflinePlugin = require(process.env.OFFLINE_PLUGIN_ROOT);
 const CopyWebpackPlugin = require('copy-webpack-plugin');
-const DefinePlugin = require('webpack').DefinePlugin;
+const DefinePlugin = require('webpack/lib/DefinePlugin');
```

Comment:
This works for webpack 2+, right?

Reply:
- [v4.1.0](https://github.com/webpack/webpack/blob/v4.1.0/lib/DefinePlugin.js)
- [v3.11.0](https://github.com/webpack/webpack/blob/v3.11.0/lib/DefinePlugin.js)
- [v2.7.0](https://github.com/webpack/webpack/blob/v2.7.0/lib/DefinePlugin.js)

👍